packages/browser/rollup.config.js

```diff
@@ -43,9 +43,7 @@ const plugins = [
     include: ['*.ts+(|x)', '**/*.ts+(|x)', '../**/*.ts+(|x)'],
   }),
   resolve({
-    module: true,
-    browser: false,
-    modulesOnly: false,
+    mainFields: ['module'],
```

Comment:
Is this the equivalent of what we had before?

Reply:
Yup, they updated the config.